googleapi/googleapi.go

```diff
@@ -259,6 +259,20 @@ func ChunkSize(size int) MediaOption {
 	return chunkSizeOption(size)
 }
 
+type chunkTransferTimeoutOption time.Duration
+
+func (cd chunkTransferTimeoutOption) setOptions(o *MediaOptions) {
+	o.ChunkTransferTimeout = time.Duration(cd)
+}
+
+// ChunkTransferTimeout returns a MediaOption which sets a per-chunk
+// transfer timeout. If a single chunk has been attempting to upload for longer than
```

Comment:
Clarify: `// transfer timeout for resumable uploads.`

Reply:
Done